src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -896,6 +909,62 @@ def test_aks_upgrade_node_image_only_nodepool(self, resource_group, resource_gro
             self.check('provisioningState', 'UpgradingNodeImageVersion')
         ])
 
+    @live_only() # live only is required for test environment setup like `az login`
```

Comment:
May I ask why `az login` is required for these tests in test environment setup? What's the problem with using the current subscription?

Reply:
Let me help explain. I think the comment here is somewhat misunderstanding. This test case needs to be executed in live mode. Without the @live_only tag, when check-in CI is running in recording mode, if the recording file corresponding to this case is not found, this case will be executed in live mode. However, az login is not configured in CI. In turn, the test will fail, prompting that az login is required.

Reply:
Maybe just keep the @live_only tag and delete the confusing comment.

Reply:
Per @FumingZhang 's explanation, I don't think it's confusing or misunderstanding. tests like this require live_only to setup the environment, and `az login` is one of the steps. 
I guess somebody else hit the same issue requiring `az login`, and add a comment like this but introduced grammatical mistake, _without live only fails with need az login_

Reply:
Actually, in check-in CI, the pipeline uses the subscription of our team to run the test, so no additional login is required. Could you show me the information of the error report?

Reply:
This is the pipeline build that asks for `az login`.
https://msazure.visualstudio.com/09706533-03bf-4b43-9a9b-b49c75429646/_apis/build/builds/44691106/logs/12

Reply:
After reading through the doc, I found the comment is a just a solution for a fix, but not really suitable, it doesn't have to be live_only test, but we need your help to explain why it requires `az login` without live_only decorator

Reply:
Removed all the misleading comments for live_only decorator.

Reply:
@zhoxing-ms please merge this PR when you think you have nowhere to change. Thanks.

Reply:
Ha, got what you mean. Let me upload the YAML file then we can remove live only. Thanks for pointing this out.

Reply:
Added generated recording file.